open-metadata-resources/open-metadata-archives/design-model-archives/src/main/java/org/odpi/openmetadata/archiveutilities/designmodels/owlcanonicalglossarymodel/CanonicalGlossaryOwlArchiveWriter.java

```diff
@@ -0,0 +1,84 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.archiveutilities.designmodels.owlcanonicalglossarymodel;
+
+import org.apache.log4j.varia.NullAppender;
+import org.odpi.openmetadata.repositoryservices.archiveutilities.OMRSArchiveWriter;
+
+
+/**
+ * CanonicalGlossaryOwlArchiveWriter creates a physical open metadata archive file for the data model and glossary
+ * content.
+ */
+public class CanonicalGlossaryOwlArchiveWriter extends OMRSArchiveWriter
+{
+    private  String openMetadataArchiveFileName;
+
+    private String modelLocation;
+
+
+    /**
+     * Default constructor
+     *
+     * @param modelLocation directory name for the model's JSON-LD files.
+     */
+    CanonicalGlossaryOwlArchiveWriter(String modelLocation)
```

Comment:
Should this name be more specific as only works for industry models ?

Reply:
We hope that others in the community will adopt this way of modelling if it suits them, there is nothing in the format that is intended to be IBM specific.